Missing oc-utils.json file when github action run

I follow the example provided on readme in your repository but when GitHub action runs this action it throw me these errors.

```
##[error]Error: ENOENT: no such file or directory, open '/Users/runner/runners/2.164.0/work/gh-action-test/gh-action-test/oc-utils.json'
##[error]Node run failed with exit code 1
```

I think the error happens because it can't find a URL link to download the client so I try to copy oc-utils.json to my repository then GitHub action can run all steps. I think you are missing the oc-utils.json file within your script. Please investigate and fix it. 

Thanks for your awesome actions.

--- a/README.md
+++ b/README.md
@@ -51,7 +51,7 @@ jobs:
     steps:
     - uses: actions/checkout@v1
     - name: OpenShift Action
-      uses: redhat-developer/openshift-actions@v1.0
+      uses: redhat-developer/openshift-actions@v1.1
       with:
         version: 'latest'
         openshift_server_url: ${{ secrets.OPENSHIFT_SERVER_URL }}

